targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_062_BLE/device/system_psoc6_cm4.c

```diff
@@ -277,13 +281,22 @@ void SystemInit(void)
 *******************************************************************************/
 void mbed_sdk_init(void)
 {
+#if !defined(COMPONENT_SPM_MAILBOX)
+    /* Disable global interrupts */
+    __disable_irq();
+#endif
+
     /* Initialize shared resource manager */
     cy_srm_initialize();
 
     /* Initialize system and clocks. */
     /* Placed here as it must be done after proper LIBC initialization. */
     SystemInit();
 
+#if defined(COMPONENT_SPM_MAILBOX)
```

Comment:
please remove this code until you actually add the BLE kit as a PSA target

Reply:
No, system_psoc6_cm4.c is deployed from internal PDL development repo, there is a desire to keep the file identical across supported PSoC6 BSPs. It is put into the BSP target directories instead of psoc6pdl directory to better align with CMSIS template (the file can changed when creating custom BSPs).